common-utils/src/common/deployment/build_image.sh

```diff
@@ -9,4 +9,4 @@ DOCKERFILE_PATH=${2}
 PYTHON_VERSION=${3}
 DOCKER_BUILD_CONTEXT=${4}
 
-docker build -t ${IMAGE_NAME} -f ${DOCKERFILE_PATH} --build-arg PYTHON_VERSION=${PYTHON_VERSION} ${DOCKER_BUILD_CONTEXT}
\ No newline at end of file
+docker build -t ${IMAGE_NAME} -f ${DOCKERFILE_PATH} --platform linux/amd64 --build-arg PYTHON_VERSION=${PYTHON_VERSION} ${DOCKER_BUILD_CONTEXT}
```

Comment:
adding the platform arg for mac support.  This means we can run this in build and M1 macs without any issues.